packages/mobile/src/components/Feed/Interacts/PostInteracts.tsx

```diff
@@ -241,7 +241,7 @@ export const LikeButton = memo(
       <>
         <ErrorNotification error={postEmojiError || removeEmojiError} />
         <IconButton
-          icon={<OpenHeart />}
+          icon={isLiked ? <SolidHeart color={Colors.red[500]} /> : <OpenHeart />}
```

Comment:
That's not what we agreed upon I'm afraid; It would show whatever you reacted with, being heart or any other emoji:

https://youfoundation.slack.com/archives/C03L659QNAC/p1729156465144279?thread_ts=1729148448.321699&cid=C03L659QNAC